scripts/pipeline.py

```diff
@@ -0,0 +1,15 @@
+from scripts.task import Task
+
+
+class Pipeline(object):
+    def __init__(self):
+        self._commands = list()
+
```

Comment:
will it make sense to make it a class `@property`?

Reply:
commands are the list of tasks to be executed as part of pipeline. Doesn't make much sense to make it a property